src/AccessToken.php

```diff
@@ -421,7 +430,12 @@ private function loadPhpsecPublicKey(string $modulus, string $exponent): string
                 $exponent
             ]), 256),
         ]);
-        return $key->toString('PKCS8');
+        // Ensure the key is correctly initialized
+        $formattedPublicKey = $key->toString('PKCS8');
+        if (is_string($formattedPublicKey)) {
+            return $formattedPublicKey;
+        }
```

Comment:
I've tracked this down to the latest version of PHPseclib. When we use `phpseclib/phpseclib: 3.0.34`, this lint error does not happen:

```
composer require --dev "phpseclib/phpseclib:3.0.34"
```

Reply:
This is exactly what I mentioned above. https://github.com/googleapis/google-auth-library-php/pull/513#issuecomment-1874447071